_dashboards/admin-ui-index/datastream.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Data streams management
+parent: Index and snapshot management in OpenSearch Dashboards
+nav_order: 20
+---
+
+# Data streams management
+Introduced 2.6
+{: .label .label-purple }
+
+In OpenSearch Dashboards, the **Index Management** section allows you to view and manage [data streams]({{site.url}}{{site.baseurl}}/opensearch/data-streams/).
+
+![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams1.png)
+
+## View data streams
+
+To view data streams and their health status, choose **Data Streams** under **Index management**.
+
+The following are the three data streams health statuses:
+
+- Green: All primary and replica shards are assigned.
+- Yellow: At least one replica shard is not assigned.
```

Comment:
An extra space/line is showing here when I build it locally.

Reply:
yeah, it's pretty weird... I'm having @kolchfa-aws take a look

Reply:
@ariamarble Remove the tab before the image to fix the spacing. Added a suggestion below.